pom.xml

```diff
@@ -393,6 +385,7 @@
                   </bind>
                 </volumes>
                 <env>
+                  <CANTALOUPE_LOG_APPLICATION_LEVEL>info</CANTALOUPE_LOG_APPLICATION_LEVEL>
```

Comment:
Is this what fixes the startup timeout?

Reply:
Yes, Cantaloupe changed its default logging level and we were using the INFO log to match for an indicator of success.